lib/versioner/simple.go

```diff
@@ -75,6 +80,50 @@ func (v simple) Restore(filepath string, versionTime time.Time) error {
 	return restoreFile(v.copyRangeMethod, v.versionsFs, v.folderFs, filepath, versionTime, TagFilename)
 }
 
-func (v simple) Clean(_ context.Context) error {
+func (v simple) Clean(ctx context.Context) error {
```

Comment:
Is there a lot of common code between this function and the one in trashcan versioner?

(Sorry, on the phone hard to check myself).

If yes, perhaps extracting common logic into a shared utility function would be beneficial.

Reply:
No. I always thought golang is a language of copy and pasting, but I guess inheritance is not necessary to be DRY. I will change.

Reply:
lol I mean yes you right